### PR TITLE
Feat/int 953 reinsurance button payment ps17

### PR DESCRIPTION
**Pull request by Benjamin-Freoua-Alma**

### CODE REVIEW

**Review by syjust-alma — COMMENTED** [review]
please add default font family as requested by Sonar (this will prevent display's issues if `Eina04` is not well downloaded by the browser for example.

**Comment by Benjamin-Freoua-Alma** [comment]
> please add default font family as requested by Sonar (this will prevent display's issues if `Eina04` is not well downloaded by the browser for example.

Done :)

**Review by syjust-alma — APPROVED** [review]

File: alma/lib/Forms/FeaturesAdminFormBuilder.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+/**
+ * Class FeaturesAdminFormBuilder
+ */
+class FeaturesAdminFormBuilder extends AbstractAlmaAdminFormBuilder
+{
+    const ALMA_ACTIVATE_REINSURANCE = 'ALMA_ACTIVATE_REINSURANCE';
+
+    protected function configForm()
```
Comment (ClaireBonacheraAlma) [inline alma/lib/Forms/FeaturesAdminFormBuilder.php:38]: maybe you can add functions doccomment

File: alma/lib/Utils/Settings.php [hunk 2]
```diff
@@ -135,6 +136,11 @@ public static function canLog()
         return (bool) (int) self::get('ALMA_ACTIVATE_LOGGING', false);
     }
 
+    public static function showReinsurance()
```
Comment (ClaireBonacheraAlma) [inline alma/lib/Utils/Settings.php:139]: doccomment?

**Review by ClaireBonacheraAlma — APPROVED** [review]

**Pull request closed without merging**